### PR TITLE
Pharaoh Sceptre and Crystal Teleport

### PR DESCRIPTION
**Pull request by vitorebom**
Added per issues #1  and #2

### CODE REVIEW

File: src/main/java/io/ryoung/menuswapperextended/MenuSwapperConfig.java [hunk 1]
```diff
@@ -181,4 +181,22 @@ default boolean swapConsCape()
...
+		return true;
+	}
+	@ConfigItem(
+			keyName = "swapPharaohSceptreLeftClick",
+			name = "Pharaoh's Sceptre",
+			description = "Change the left-click option on Pharaoh Sceptre"
+	)
+	default PharaohSceptreMode swapPharaohSceptreLeftClick()
+	{
+		return PharaohSceptreMode.JALSAVRAH;
```
Comment (raiyni) [inline src/main/java/io/ryoung/menuswapperextended/MenuSwapperConfig.java:200]: This should default to whatever the default left click is for the sceptre.

Comment (vitorebom) [reply src/main/java/io/ryoung/menuswapperextended/MenuSwapperConfig.java:200]: [https://imgur.com/2Dee9bP](url) Jalsavrah is the default option

Comment (vitorebom) [reply src/main/java/io/ryoung/menuswapperextended/MenuSwapperConfig.java:200]: [https://i.imgur.com/tGz974S.png](url) Lletya is default for teleport crystal

Comment (raiyni) [reply src/main/java/io/ryoung/menuswapperextended/MenuSwapperConfig.java:200]: Those are the defaults without these options on?

Comment (vitorebom) [reply src/main/java/io/ryoung/menuswapperextended/MenuSwapperConfig.java:200]: Correct

File: src/main/java/io/ryoung/menuswapperextended/PharaohSceptreMode.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright (c) 2018, Henry Darnell <https://github.com/hjdarnel>
```
Comment (raiyni) [inline src/main/java/io/ryoung/menuswapperextended/PharaohSceptreMode.java:2]: This needs to be updated to you

File: src/main/java/io/ryoung/menuswapperextended/MenuSwapperConfig.java [hunk 3]
```diff
@@ -181,4 +181,22 @@ default boolean swapConsCape()
...
 		return true;
 	}
+	@ConfigItem(
+			keyName = "swapTeleCrystal",
+			name = "Teleport Crystal",
+			description = "Swaps the Crystal Seed Teleport to Prifddinas"
+	)
+	default boolean swapTeleCrystal()
+	{
+		return true;
```
Comment (raiyni) [inline src/main/java/io/ryoung/menuswapperextended/MenuSwapperConfig.java:191]: Default this to false. After thinking about it, I want to start making options off by default unless they are 100% whatever does by default.

File: src/main/java/io/ryoung/menuswapperextended/MenuSwapperPlugin.java [hunk 4]
```diff
@@ -220,6 +220,14 @@ else if (!shiftHeld && config.swapConsCape() && (option.equals("teleport")) && (
 		{
 			swap("tele to poh", option, target, index);
 		}
+		else if (!shiftHeld && config.swapTeleCrsytal() && option.equals("lletya"))
+		{
+			swap("prifddinas", option, target, index);
+		}
+		else if (!shiftHeld && target.startsWith("Pharaoh's sceptre") && option.equals("Jalsavrah"))
```
Comment (raiyni) [inline src/main/java/io/ryoung/menuswapperextended/MenuSwapperPlugin.java:227]: Lower case these, because options get converted to lowercase at the top

**Pull request merged**